modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/main.es.js

```diff
@@ -187,9 +187,14 @@ class Form extends Component {
 			)
 		);
 
+		const shareURLButton = document.querySelector(
+			'.lfr-ddm-share-url-button'
+		);
+
 		if (showPublishAlert) {
 			if (published) {
 				this._showPublishedAlert(this._createFormURL());
+				shareURLButton.removeAttribute('title');
```

Comment:
@alinedoleron, don't you think that it's worth to keep the tooltip even with the published form? Did Felipe advise you to not keep it?

Reply:
Hi, @natocesarrego 
Yes, Felipe said there's no need to show the tooltip when Share button is enabled. So I removed the title attribute because if I keep it without content it would show an empty tooltip.